src/LoginOrApp.js

```diff
@@ -18,16 +18,67 @@ class LoginOrAppComponent extends Component {
   }
 
   componentDidMount() {
-    this.props.fetchBootstrap(this.props.sessionState);
+    const dashboardName = this.getDashboardName();
+    this.props.fetchBootstrap(this.props.sessionState, dashboardName);
   }
 
+  getDashboardName = () => {
+    // gets dashboard name from url as defined in /admin/lizard_nxt/clientconfiguration/ "client slug" field
+    // the url should be in format of /dashboard/<dashboard_name>/
+
+    // if no dashboard name is given this function should return undefined
+    // this is required to create the basename of the JSX Router component (see in this file)
+
+    //  if a dashboard name of undefined is returned then the default name 'dashboard' will be used by function getBootstrap (in file actions.js).
+    // when no dashboard is in the url at all this function should return undefined. this should only be the case on dev
+
+    // if dashboardname = 'full' we assume it is not really the dashboardname but the url path to a full tile
+    // this rule should ensures that the url /dashboard/full/ returns  undefined
+
+    // examples:
+    // /dashboard/my_name/ -> my_name
+    // /dashboard/my_name -> my_name
+    // /dashboard/dashboard/ -> dashboard
+    // /dashboard/dashboard -> dashboard
+    // /dashboard/full/1 -> undefined
+    // /dashboard/ -> undefined
+    // /dashboard -> undefined
+    // / -> undefined
+
+    // split on first occurence of dashboard as in:
+    // https://stackoverflow.com/questions/4607745/split-string-only-on-first-instance-of-specified-character
+    const urlPostDashboard = window.location.href.split(/dashboard(.+)/)[1];
+    // if there was no dashboard in the url, should only happen in dev
+    if (!urlPostDashboard) {
+      return undefined;
+    }
+    const dashboardName = urlPostDashboard.split("/")[1];
```

Comment:
Het lastige is dat iets als /dashboardmetnogwatmeer/naam/ nu ook werkt. Zou de eerste / al  in de vorige .split kunnen zitten, zodat hier [0] genomen kan worden?

Dus

    const urlPostDashboard = window.location.href.split(/dashboard\//)[1];
    const dashboardName = urlPostDashboard.split("/")[0];

Reply:
Ja dit kan denk ik en zou bovenstaande comment waarschijnlijk ook oplossen. Dit ga ik proberen.

Reply:
Hi Remco,
Dit heb ik doorgevoerd. Kon hierdoor direct ook de verwarrende regex hierboven veranderen in een simpelere string.split.